Update backup config docs to mention backoffLimit (#3044)
codenvy/che-docs: modules/administration-guide/pages/devworkspace-backup.adoc

* `enable`: Set to `true` to enable the backup job or `false` to disable it. The default value is `false`.
* `schedule`: A Cron expression that defines the backup frequency. The default value is `"0 1 * * *"`.

> `backoffLimit`:: Specifies the number of retries before a backup job is marked as failed. The default value is `1`.

Reviewer: ```suggestion
* `backoffLimit`:: Specifies the number of retries before a backup job is marked as failed. The default value is `1`.
```

> * `backoffLimit`: The value of the backup job's `spec.backoffLimit`, which specifies the number of retries before marking this job failed. The default value for the backups jobs is `1`.

Reviewer: ```suggestion
`backoffLimit`:: Specifies the number of retries before a backup job is marked as failed. The default value is `1`.
```
